### DISCUSSION TITLE
Is there any prowler scan rule that scan on Aurora mysql?

### DISCUSSION THREAD

**Discussion by NMuee** [prowler-cloud/prowler#1455] [Q&A] [answered]
As asked in the title. 
Thank you for your help.

**Comment by jfagoagas** [comment, marked as answer]
Hi @NMuee, Prowler doesn't have any rule related to RDS Aurora for the MySQL engine, the only one regarding MySQL is the check which covers 3306 port [here](https://github.com/prowler-cloud/prowler/blob/1e422f20aa02410147fcb3a2ce1822f78fa22d87/checks/check_extra750).

Thanks for using Prowler!
